python/lsst/ts/xml/data/sal_interfaces/MTMount/MTMount_Telemetry.xml

```diff
@@ -9,7 +9,6 @@
   <SALTelemetry>
     <Subsystem>MTMount</Subsystem>
     <EFDB_Topic>MTMount_azimuth</EFDB_Topic>
-    <Description>Azimuth axis detail</Description>
```

Comment:
Why did you remove this and all other topic descriptions? There is a unit test that checks if all topics have descriptions so that should fail now. See https://github.com/lsst-ts/ts_xml/blob/develop/tests/test_TopicDescription.py#L20-L21 why it doesn't.